plugins/optimization-detective/tests/test-uninstall.php

```diff
@@ -24,7 +24,7 @@ public static function set_up_before_class(): void {
 	 * Load uninstall.php.
 	 */
 	private function require_uninstall(): void {
-		require __DIR__ . '/../../../plugins/optimization-detective/uninstall.php';
+		require TESTS_REPO_ROOT_DIR . '/plugins/optimization-detective/uninstall.php';
```

Comment:
See my other comment:
```suggestion
		require TESTS_PLUGIN_DIR . '/uninstall.php';
```